README.md

```diff
@@ -11,25 +11,50 @@ any installed credential provider on the device.
 This repository contains the reference implementation of the OpenYOLO protocol,
 and the standard API and Service Provider Interface (SPI).
 
-OpenYOLO is inspired by, and ultimately intended to replace, Google's
-[Smart Lock for Passwords][yolo] API, which is
-internally known as YOLO (You Only Login Once).
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/445775/30930082-60d21ace-a375-11e7-90cf-cc75fe5747fd.gif"/>
+  <p><em>Example of auto sign-in</em></p>
+</div>
 
-## OpenYOLO is *not* production ready
+## Project Status
 
-OpenYOLO is currently in the *experimental* stage; both the specification and
-reference implementation are *unstable*. We do not recommend that it be used
-in production Android apps. At such a time that OpenYOLO is considered stable
-and ready for production use, an official announcement shall be made and both
-the specification and reference implementation will be promoted to v1.0.0.
+OpenYOLO has reached beta production ready stage and can be integrated into production applications.
+There are currently several production implementation of credential providers including: 
 
-## OpenYOLO Specification
+| <img src="https://user-images.githubusercontent.com/445775/31100963-59c51db2-a780-11e7-87f4-e7ab2750b92a.png" width="200" height="200" /> | <img src="https://user-images.githubusercontent.com/445775/31101083-0cc4fd92-a781-11e7-95fe-9f9e9fadae70.png" width="200" height="200" /> | <img src="https://user-images.githubusercontent.com/445775/31101050-eeb9966e-a780-11e7-96ae-614f765f3f12.png" width="200" height="200" /> |
+|:---:|:---:|:---:|
+| 1Password | Dashlane | Google Smart Lock |
 
-The OpenYOLO for Android specification can be found 
-[here](https://spec.openyolo.org/openyolo-android-spec.html). This is
-currently under review by the 
-[Account Chooser and OpenYOLO Working Group](http://openid.net/wg/ac/),
-with the intention for it to become an implementor's draft by Q3 2017,
-and reach full standardization by Q4 2017.
+The OpenYOLO for Android specification can be found [here](http://openid.net/specs/openyolo-android-03.html).
+This is currently under review by the [Account Chooser and OpenYOLO Working Group](http://openid.net/wg/ac/),
+and has reached implementor's draft with the intention for it to reach full standardization in the
+near future.
+
+## Getting Started as a Client
+
+[ ![Download](https://api.bintray.com/packages/openid/net.openid/openyolo-api/images/download.svg?version=0.2.0) ](https://bintray.com/openid/net.openid/openyolo-api/0.2.0/link)
+
+The open-source client library for the OpenYOLO for Android protocol lives in the ```api```
```

Comment:
You can omit "open-source" here, as it's redundant info.

Reply:
Done.